clients/admin-ui/src/features/system/hooks/useSystemFormTabs.tsx

```diff
@@ -219,9 +220,13 @@ const useSystemFormTabs = ({
       content: activeSystem ? (
         <Box width={{ base: "100%", lg: "70%" }}>
           <Box px={6} paddingBottom={2}>
-            <Text fontSize="sm" lineHeight={5} fontWeight="medium">
-              Integrations are used to process privacy requests for access,
-              erasure, portability, rectification, and consent.
+            <Text fontSize="sm" lineHeight={5}>
+              Add an integration to start managing privacy requests and consent.
+              Navigate{" "}
+              <Link href={INTEGRATION_MANAGEMENT_ROUTE} color="purple.500">
+                here
+              </Link>{" "}
+              to set up monitoring on databases.
```

Comment:
"here" is generally frowned upon as link text. Can we link something more semantic?
```suggestion
              Add an integration to start managing privacy requests and consent
              <Link href={INTEGRATION_MANAGEMENT_ROUTE} color="purple.500">
                Visit Integration Management
              </Link>{" "}
              to set up monitoring on databases.
```